instrumentation/net/http/otelhttp/transport_test.go

```diff
@@ -243,3 +246,101 @@ func TestTransportUsesFormatter(t *testing.T) {
 	}
 
 }
+
+func TestTransportErrorStatus(t *testing.T) {
+	// Prepare tracing stuff.
+	prop := propagation.TraceContext{}
```

Comment:
I guess this can be removed as it is not used

Reply:
Right! done in [`ed15ce8` (#950)](https://github.com/open-telemetry/opentelemetry-go-contrib/pull/950/commits/ed15ce8d81b07ac9aedd98d29aa8455741a1464d)